ocpmodels/trainers/base_trainer.py

```diff
@@ -360,7 +346,9 @@ def train(self, max_epochs=None, return_metrics=False):
             return {
                 "training_loss": float(self.meter.loss.global_avg),
                 "training_mae": float(
-                    self.meter.meters["binding energy/mae"].global_avg
+                    self.meter.meters[
+                        self.config["task"]["labels"][0] + "/mae"
```

Comment:
Probably don't want to hardcode `"/mae` either, `self.config["task"]["metric"]` instead.

Reply:
Good catch! Thanks!